keyboards/keycapsss/kimiko/rev2/keymaps/default/rules.mk

```diff
@@ -0,0 +1,10 @@
+OLED_ENABLE = yes
+
+ENCODER_ENABLE = yes       # Enables the use of one or more encoders
+ENCODER_MAP_ENABLE = yes
+
+RGBLIGHT_ENABLE = no      # Enable keyboard 'old' RGB lightning
+RGB_MATRIX_ENABLE = yes   # Enable keyboard RGB Matrix lightning
```

Comment:
there's no rgb_matrix information for this keyboard (`kimiko/rev2`), enabling rgb_matrix has no effect.